packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts

```diff
@@ -47,7 +47,7 @@ export class ChainHistoryBuilder {
 
   public async queryTransactionInputsByHashes(hashes: Cardano.TransactionId[], collateral = false): Promise<TxInput[]> {
     const byteHashes = hashes.map((hash) => hexStringToBuffer(hash.toString()));
-    this.#logger.debug(`About to find inputs (collateral: ${collateral}) for transactions:`, byteHashes);
+    this.#logger.debug(`About to find inputs (collateral: ${collateral}) for transactions:`, hashes);
```

Comment:
Since the log line no longer requires the variable set on the above line, it would be better to move this above it to line 49. The same feedback applies to all log lines where this change was made.

Reply:
ed6416dc7 should solve this